spec/features/favorites/index_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe "Favorite a Pet" do
```

Comment:
This will need to be redone but just made this favorite index to pass the application form